### PR TITLE
rm evm compatibility + make it compatible with rust verifier

### PR DESCRIPTION
**Pull request by Elli610**
**breaking change**

Works with rust verifier: https://github.com/Cypher-Laboratory/rust_RS_verifier/pull/2

### CODE REVIEW

File: src/utils/randomNumbers.ts [hunk 1]
```diff
@@ -19,6 +19,9 @@ if (
  * @returns the random bigint
  */
 export function randomBigint(max: bigint): bigint {
+  return BigInt(
```
Comment (LeJamon) [inline src/utils/randomNumbers.ts:22]: why ?

File: src/ringSignature.ts [hunk 2]
```diff
@@ -668,26 +604,14 @@ export class RingSignature {
...
-
-      return {
-        cee: modulo(BigInt("0x" + hash(hashContent, config)), N),
-        evmWitnesses: [],
-      };
+
+      const hashContent = (serializedRing as (bigint | string)[])
+        .concat([messageDigest])
+        .concat([alphaG.serialize()])
+        .concat(/*BigInt(*/params.customMapped.mult(params.alpha).serialize()/*.toEthAddress())*/);
```
Comment (LeJamon) [inline src/ringSignature.ts]: comments to remove ?

File: src/ringSignature.ts [hunk 3]
```diff
@@ -707,41 +631,21 @@ export class RingSignature {
...
+        .concat([messageDigest])
+        .concat([point.serialize()])
         .concat(
-          BigInt(
-            mapped
-              .mult(params.previousR)
-              .add(params.keyImage.mult(params.previousC))
-              .toEthAddress(),
-          ),
+          // BigInt(
```
Comment (LeJamon) [inline src/ringSignature.ts:636]: comments to remove

File: src/ringSignature.ts [hunk 4]
```diff
@@ -707,41 +631,21 @@ export class RingSignature {
...
+          mapped
+            .mult(params.previousR)
+            .add(params.keyImage.mult(params.previousC))
+            .serialize(),
+          // .toEthAddress(),
```
Comment (LeJamon) [inline src/ringSignature.ts:641]: comments to remove

File: src/ringSignature.ts [hunk 5]
```diff
@@ -707,41 +631,21 @@ export class RingSignature {
...
+          // ),
```
Comment (LeJamon) [inline src/ringSignature.ts:642]: comments to remove

File: src/ringSignature.ts [hunk 6]
```diff
@@ -668,26 +597,17 @@ export class RingSignature {
...
-            ? BigInt(alphaG.toEthAddress())
-            : BigInt("0x" + alphaG.serialize()),
-        ])
-        .concat(BigInt(params.customMapped.mult(params.alpha).toEthAddress()));
-
-      return {
-        cee: modulo(BigInt("0x" + hash(hashContent, config)), N),
-        evmWitnesses: [],
-      };
+          /*BigInt(*/ params.customMapped
```
Comment (LeJamon) [inline src/ringSignature.ts:605]: comments to remove

File: src/ringSignature.ts [hunk 7]
```diff
@@ -668,26 +597,17 @@ export class RingSignature {
...
+            .mult(params.alpha)
+            .serialize() /*.toEthAddress())*/,
```
Comment (LeJamon) [inline src/ringSignature.ts:607]: comments to remove

**Review by LeJamon — CHANGES_REQUESTED** [review]
Comments to remove and strange thing in randomNumbers.ts

File: src/utils/randomNumbers.ts (hunk 1 above)
Comment (Elli610) [reply src/utils/randomNumbers.ts:22]: used for debug (it would always give the same sig )

File: src/ringSignature.ts (hunk 2 above)
Comment (Elli610) [reply src/ringSignature.ts]: done 
https://github.com/Cypher-Laboratory/Alice-s-Ring-LSAG-TS/commit/c58210f3545346cfb6fe35998ce0a48bb9e87ab1

File: src/ringSignature.ts (hunk 3 above)
Comment (Elli610) [reply src/ringSignature.ts]: done 
https://github.com/Cypher-Laboratory/Alice-s-Ring-LSAG-TS/commit/c58210f3545346cfb6fe35998ce0a48bb9e87ab1

File: src/utils/randomNumbers.ts (hunk 1 above)
Comment (Elli610) [reply src/utils/randomNumbers.ts]: fixed 
https://github.com/Cypher-Laboratory/Alice-s-Ring-LSAG-TS/commit/c58210f3545346cfb6fe35998ce0a48bb9e87ab1

File: src/ringSignature.ts (hunk 4 above)
Comment (Elli610) [reply src/ringSignature.ts]: done 
https://github.com/Cypher-Laboratory/Alice-s-Ring-LSAG-TS/commit/c58210f3545346cfb6fe35998ce0a48bb9e87ab1

File: src/ringSignature.ts (hunk 5 above)
Comment (Elli610) [reply src/ringSignature.ts]: done 
https://github.com/Cypher-Laboratory/Alice-s-Ring-LSAG-TS/commit/c58210f3545346cfb6fe35998ce0a48bb9e87ab1

File: src/ringSignature.ts (hunk 6 above)
Comment (Elli610) [reply src/ringSignature.ts]: done 
https://github.com/Cypher-Laboratory/Alice-s-Ring-LSAG-TS/commit/c58210f3545346cfb6fe35998ce0a48bb9e87ab1

File: src/ringSignature.ts (hunk 7 above)
Comment (Elli610) [reply src/ringSignature.ts]: done 
https://github.com/Cypher-Laboratory/Alice-s-Ring-LSAG-TS/commit/c58210f3545346cfb6fe35998ce0a48bb9e87ab1

**Review by LeJamon — APPROVED** [review]
lgtm

**Pull request merged**